bool left_ls
bool right_ls
float32 bed_position # In MM
float32 bed_length   # In MM